src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -39,23 +65,21 @@ internal abstract partial class AbstractObjectBrowserLibraryManager : AbstractLi
         private object _classMemberGate = new object();
 
         private readonly IEnumerable<Lazy<IStreamingFindUsagesPresenter>> _streamingPresenters;
+        private readonly Lazy<IComponentModel> _componentModel;
 
         protected AbstractObjectBrowserLibraryManager(
```

Comment:
❗️ At the time this constructor is called, `VisualStudioWorkspace` has already been obtained by the caller. There is no value in lazily accessing it; you can make it a constructor argument and have it for free.